src/components/interaction-designer/block-types/ConsoleIO_PrintBlock.vue

```diff
@@ -33,30 +33,30 @@ import { IPrintBlock } from '@floip/flow-runner/src/model/block/IPrintBlock'
 import { IResourceDefinition } from '@floip/flow-runner/src/domain/IResourceResolver'
 
 import PrintStore, { BLOCK_TYPE } from '@/store/flow/block-types/ConsoleIO_PrintBlockStore'
-import lang from '@/lib/filters/lang'
+import { Lang } from '@/lib/filters/lang'
 import { createDefaultBlockTypeInstallerFor } from '@/store/builder'
 import ResourceEditor from '../resource-editors/ResourceEditor.vue'
 import BlockNameEditor from '../block-editors/NameEditor.vue'
 import BlockLabelEditor from '../block-editors/LabelEditor.vue'
 import BlockSemanticLabelEditor from '../block-editors/SemanticLabelEditor.vue'
 import FirstBlockEditorButton from '../flow-editors/FirstBlockEditorButton.vue'
 import BlockId from '../block-editors/BlockId.vue'
+import { mixins } from 'vue-class-component'
 
 const flowVuexNamespace = namespace('flow')
 const builderVuexNamespace = namespace('builder')
 
-  @Component<any>({
-    components: {
-      ResourceEditor,
-      BlockNameEditor,
-      BlockLabelEditor,
-      BlockSemanticLabelEditor,
-      FirstBlockEditorButton,
-      BlockId,
-    },
-    mixins: [lang],
-  })
-class ConsoleIO_PrintBlock extends Vue {
+@Component({
+  components: {
+    ResourceEditor,
+    BlockNameEditor,
+    BlockLabelEditor,
+    BlockSemanticLabelEditor,
+    FirstBlockEditorButton,
+    BlockId,
+  },
+})
+class ConsoleIO_PrintBlock extends mixins(Lang) {
```

Comment:
Nice. I almost did this in my branch but decided not to mess with `@/lib/filters/lang`. Good to finally get it done. I notice a few places we will use the old `mixins: [lang]`. Should we update those?

Reply:
Changing the old approach `mixins: [lang]` requires we turn the component into full vue class, which could be time consuming and is not very important for now. I'd propose we create a ticket for Beta version to turn all component into class based.

Reply:
Sounds good. Let's leave it